.readthedocs.yaml

```diff
@@ -0,0 +1,21 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+build:
+  os: ubuntu-22.04
+  apt_packages:
+    - swig
+    - softhsm2
```

Comment:
`swig` and `softhsm2` are needed for `pykcs11`, which we install in requirements-docs.txt via `requirements-pinned.txt`. 

We can probably shuffle requirements* files so that we don't need to install any optional requirements.

Reply:
suggestion makes sense but if rtd is fine with us installing random software... I suppose this is ok